netket/jax/__init__.py

```diff
@@ -29,6 +29,7 @@
     tree_axpy,
     tree_to_real,
     compose,
+    logsumexp,
```

Comment:
```suggestion
```

Reply:
If we need to ship your version for now, why not put it as `nkjax.logsumexp`?

Reply:
Because I'd rather do this in another PR.
For example, do we put it in nkjax or in `nkjax.scipy.special.logsumexp` ?
It has implications with respect to how we will version nkjax